lib/hardware/modules/div/iob_div_subshift_signed/iob_div_subshift_signed.py

```diff
@@ -3,11 +3,219 @@ def setup(py_params_dict):
+                "descr": "Input port",
+                "signals": [
+                    {
+                        "name": "dividend",
+                        "width": "DATA_W",
+                        "direction": "input",
+                    },
+                ],
+            },
+            {
+                "name": "divisor_i",
+                "descr": "Input port",
+                "signals": [
+                    {
+                        "name": "divisor",
+                        "width": "DATA_W",
+                        "direction": "input",
+                    },
+                ],
+            },
+            {
+                "name": "done_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "done",
+                        "width": 1,
+                        "direction": "output",
+                    },
+                ],
+            },
+            {
+                "name": "quotient_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "quotient",
+                        "width": "DATA_W",
+                        "direction": "output",
+                    },
+                ],
+            },
+            {
+                "name": "remainder_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "remainder",
+                        "width": "DATA_W",
+                        "direction": "output",
+                    },
+                ],
+            },
+        ],
+        "wires": [
+            {
+                "name": "rq",
+                "descr": "rq wire",
+                "signals": [
+                    {"name": "rq", "width": "2*DATA_W"},
+                ],
+            },
+            {
+                "name": "divisor_reg",
+                "descr": "divisor_reg wire",
+                "signals": [
+                    {"name": "divisor_reg", "width": "DATA_W"},
+                ],
+            },
+            {
+                "name": "divident_sign",
+                "descr": "divident_sign wire",
+                "signals": [
+                    {"name": "divident_sign", "width": 1},
+                ],
+            },
+            {
+                "name": "divisor_sign",
+                "descr": "divisor_sign wire",
+                "signals": [
+                    {"name": "divisor_sign", "width": 1},
+                ],
+            },
+            {
+                "name": "pc",
+                "descr": "pc wire",
+                "signals": [
+                    {"name": "pc", "width": "PC_W"},
+                ],
+            },
+            {
+                "name": "subtraend",
+                "descr": "subtraend wire",
+                "signals": [
+                    {"name": "subtraend", "width": "DATA_W"},
```

Comment:
The subtraend wire is not being driven.
In the original module, it had a wire (implicit assign) driving it:
https://github.com/IObundle/iob-soc/blob/8f10f1bd5fd583e04d524879af6ef14e0211bb2f/lib/hardware/modules/div/iob_div_subshift_signed/hardware/src/iob_div_subshift_signed.v#L25

We need to add a snippet with this assign.